.github/workflows/helm-lint-test.yaml

```
@@ -2,6 +2,7 @@ name: "Helm lint and tests"
 on:
   pull_request:
     types: [opened, synchronize, reopened]
+  workflow_dispatch:
```

I think this YAML is invalid. Should be either without the `:` or with the empty object notation
```suggestion
  workflow_dispatch: {}
```